lighthouse-core/scripts/i18n/collect-strings.js

```diff
@@ -402,66 +470,54 @@ function collectAllStringsInDir(dir) {
 
     const content = fs.readFileSync(absolutePath, 'utf8');
     const exportVars = require(absolutePath);
-    const regexMatches = UISTRINGS_REGEX.test(content);
-    const exportsUIStrings = !!exportVars.UIStrings;
-    if (!regexMatches && !exportsUIStrings) continue;
+    const regexMatch = content.match(UISTRINGS_REGEX);
+    const exportedUIStrings = exportVars.UIStrings;
 
-    if (regexMatches && !exportsUIStrings) {
-      throw new Error('UIStrings defined but not exported');
+    if (!regexMatch) {
+      if (!exportedUIStrings) continue;
+      throw new Error('UIStrings exported but no definition found');
     }
 
-    if (exportsUIStrings && !regexMatches) {
-      throw new Error('UIStrings exported but no definition found');
+    if (!exportedUIStrings) {
+      throw new Error('UIStrings defined but not exported');
     }
 
-    // @ts-ignore regex just matched
-    const justUIStrings = 'const ' + content.match(UISTRINGS_REGEX)[0];
     // just parse the UIStrings substring to avoid ES version issues, save time, etc
-    // @ts-ignore - esprima's type definition is supremely lacking
-    const ast = esprima.parse(justUIStrings, {comment: true, range: true});
-
-    for (const stmt of ast.body) {
```

Comment:
there's only ever one `stmt` (if we did our extraction correctly), so no need to loop :)